src/common/options.cc

```diff
@@ -1530,25 +1530,25 @@ std::vector<Option> get_global_options() {
 
     Option("filer_max_purge_ops", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(10)
-    .set_description(""),
+    .set_description("Max in-flight operations for purging a striped range (e.g., MDS journal)"),
 
     Option("filer_max_truncate_ops", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(128)
-    .set_description(""),
+    .set_description("Max in-flight operations for truncating/deleting a striped sequence (e.g., MDS journal)"),
 
     Option("journaler_write_head_interval", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(15)
-    .set_description(""),
+    .set_description("Frequency of journal header updates (to help bound replay time)"),
```

Comment:
imho, the unit of frequency is hertz not second. can we s/Frequency/Interval in seconds/ here?